dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassSymbolTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.testing.EqualsTester;
+import com.google.common.testing.NullPointerTester;
+import com.google.common.testing.NullPointerTester.Visibility;
+import org.junit.Test;
+
+public class ClassSymbolTest {
+
+  @Test
+  public void testClassSymbolCreation() {
+    ClassSymbol classSymbol = new ClassSymbol("java.lang.Object");
+    assertEquals("java.lang.Object", classSymbol.getClassName());
+
+    new NullPointerTester().testConstructors(ClassSymbol.class, Visibility.PACKAGE);
```

Comment:
This should be a separate test method. Common code can be pulled up into a fixture (setUp method + fields). This method might be testGetClassName

Reply:
Done (with no common code).